### ISSUE TITLE
[Request] Add Support For Creating Hybrid Torrents

### ISSUE THREAD

**Issue by zero77** [Tribler/tribler#5847]
Can you please ad support for creating hybrid torrents as a default option but, with the option to create a v1 one torrent if required.

hybrid torrent test at the bottom:
https://blog.libtorrent.org/2020/09/bittorrent-v2/

**Comment by xoriole** [comment]
Thank you for your request. We had a short discussion about supporting v2 torrents earlier and it will require quite some time. Please stay patient.

**Comment by zero77** [comment]
@xoriole 
Yes i saw the issues relating to Bittorrent v2 support, i appreciate it will be a lot of work.

However, hybrid torrents are backward compatible and are working at the moment, i have tested against v7.6.1 installed in Debian.

**Comment by drew2a** [comment]
Related to
- #5556
